### ISSUE TITLE
XFCE image *verry* sluggish

### ISSUE THREAD

**Issue by sakittelsen** [radxa-build/rock-4se#4]
Installing the XFCE image gives a *very* sluggish gui and slow responding system.
Using the cli image and then installing XFCE gives a normal and much faster system.

**Comment by sakittelsen** [comment]
I guess this could be a Debian issue, so closing the issue.

**Comment by sakittelsen** [comment]
No, this is not a Debian issue, because XFCE runs fine when installing XFCE with the CLI image.

**Comment by sakittelsen** [comment]
I think I have heard that the cpu is not running at normal full speed?

**Comment by RadxaYuntian** [comment]
I blame Rockchip packages. Anyway needs to triangulate which exactly caused this.

**Comment by sakittelsen** [comment]
Use the non-modified stock xorg package from Debian and it will work much better...
